content/docs/week02.adoc

```diff
@@ -14,6 +14,18 @@ toc: true
 
 :imagesdir: /prc2_web/
 
+== Clean tests
```

Comment:
I think FIRST principles are helpful, not sure where they fit in best
- week 1 seems pretty cramped already?
- in week 2 it might give us a nice opportunity to revisit some fundamentals like TDD and AAA ?